AppBuilder/platform/views/viewComponent/ABViewReportsManagerComponent.js

```diff
@@ -393,7 +393,6 @@ module.exports = class ABViewReportsManagerComponent extends ABViewComponent {
 
                   switch (r.type) {
                      case "date":
-                     case "datetime":
```

Comment:
Why did you remove `datetime`?

Reply:
on `view: "query"`, the `datetime` type is `date`. Previously, the`datetime` type of selection criteria did not change to `date` types.
https://docs.webix.com/api__ui.query_fields_config.html